.github/actions/pr-body-parse/action.yml

```diff
@@ -0,0 +1,37 @@
+name: PR Body Parse
+description: Parse a value from the body of the relevant PR for a specific key
+
+inputs:
+  key-regex:
+    description: "The regex for the key identifying the value to return, e.g. BRIEFCASE[-_]*REPO"
+    required: true
+  testing-pr-body:
+    description: "Override value for body of PR; only for testing."
+    required: false
+
+outputs:
+  extracted-value:
+    value: ${{ steps.parse.outputs.extracted-value }}
```

Comment:
A minor nitpick - but if we're only returning one value, is there any reason to *not* call it `value`, rather than `extracted-value`? These keys are already pretty long, and something like `${{ steps.briefcase-repo-override.outputs.value }}` reads very slightly better.

Reply:
I annotated the output with `extracted` to help provide some breadcrumbs later on. That is, when you actually reference this output, it is completely detached from this Action and is only associated with the `id` of the step that ran the action; for instance, `steps.briefcase-repo-override.outputs.extracted-value`. So, having `extracted` helps remind you that this value was parsed from an arbitrary source.

That said, is that useful information later on? maybe, maybe not, I guess. But it does make these variables even longer than they already are.